contracts/hackatom/src/contract.rs

```diff
@@ -614,6 +614,6 @@ mod tests {
             expected_hash = Sha256::digest(&expected_hash);
         }
         let work_query = query_recurse(&deps, 0, 5, contract).unwrap();
-        assert_eq!(work_query.hashed, expected_hash.to_vec().into());
+        assert_eq!(work_query.hashed, expected_hash.to_vec());
```

Comment:
This is the case I have in the rand contract as well: A query's `Binary` result is tested against binary data generated with other crates (`Vec<u8>`).